src/specs/VideoNativeComponent.ts

```diff
@@ -373,45 +369,45 @@ export interface VideoNativeProps extends ViewProps {
   onVideoTracks?: DirectEventHandler<OnVideoTracksData>; // android
 }
 
-export type VideoComponentType = HostComponent<VideoNativeProps>;
-
-export type VideoSaveData = {
-  uri: string;
-};
-
-export interface VideoManagerType {
-  save: (option: object, reactTag: number) => Promise<VideoSaveData>;
-  seek: (option: Seek, reactTag: number) => Promise<void>;
-  setPlayerPauseState: (paused: boolean, reactTag: number) => Promise<void>;
+type NativeVideoComponentType = HostComponent<VideoNativeProps>;
+type NativeVideoComponentRefType = React.ElementRef<NativeVideoComponentType>;
+
+export interface NativeVideoCommands {
+  seek: (
+    viewRef: NativeVideoComponentRefType,
+    time: Float,
+    tolerance?: Float,
+  ) => void;
```

Comment:
It seems like we need to half-ly revert prev commit - function that are command need to have first arg that is type `React.ElementRef<T>` (even if we created alias it does not work with codegen)
so we need to change `viewRef: NativeVideoComponentRefType` to `viewRef: React.ElementRef<NativeVideoComponentType>`